BUILD.bazel

```diff
@@ -6,21 +6,29 @@ load("//bazel:pkg_info.bzl", "pkg_variables", "pkg_version")
 package(default_visibility = ["//visibility:public"])
 
 platform(
-    name = "arm64-apple-darwin",
+    name = "apple-darwin-arm64",
```

Comment:
in the Bazel/LLVM world we should use `aarch64` and `x86_64` while Debian/Ubuntu calls it `arm64` and `amd64`.